test/wallet/execute_transactions_test.go

```diff
@@ -0,0 +1,326 @@
+package wallet_test
+
+import (
+	"context"
+	"math/big"
+	"strings"
+    "fmt"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+    "github.com/ethereum/go-ethereum/core/types"
+    "github.com/ethereum/go-ethereum"
+    "github.com/tokencard/ethertest"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/tokencard/contracts/v2/test/shared"
+)
+
```

Comment:
Can you run some type of formatter on the files, it seems that the indentation is a bit off